src/scss/utilities/mixins/_gradients.scss

```diff
@@ -1,6 +1,6 @@
 @charset "UTF-8";
 
-// solid gradients
+// Solid gradients
 @mixin gradient--solid-purple-lagunita {
   background: linear-gradient(91.01deg, $su-color-purple 3.46%, $su-color-lagunita 95.48%);
```

Comment:
Man, they're picky about that gradient.  Two significant digits of precision?  Gosh.